app/Common.php

```diff
@@ -36,3 +39,13 @@ function vite(string|array $path): string
 
     return $vite->links($path);
 }
+
+if (! function_exists('manager')) {
```

Comment:
I think it probably would be okay to do the manager simply and just use `new CategoryManager()`. I don't see it as being something that we'd need to worry about a singleton for, or need to override for this project, so a service wouldn't really be needed either. 

We can keep it this way, it just seems overkill, personally.

Reply:
I might be wrong here, actually. I see that having it in the Policy makes it where we could need a singleton.

Reply:
Yes, I thought so too.